bin/check_go_path

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+if [ -z "$GOPATH" ]; then
+	echo "GOPATH not set, you must have go configured properly to install ipfs"
```

Comment:
. You

Note: @whyrusleeping you use commas a _lot more_ than they should be used. In cases like this, use a period.